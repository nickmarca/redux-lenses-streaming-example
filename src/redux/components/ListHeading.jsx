import React from 'react'

function ListHeading ({ numberOfMessages }) {
    return (
        <div className="panel-block" style={{borderTop: 0}}>
            <div className="control">
                Number of messages: {numberOfMessages}
            </div>
        </div>
    );
}

export default ListHeading;