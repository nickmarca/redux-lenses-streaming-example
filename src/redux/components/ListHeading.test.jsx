import React from 'react';
import { create } from 'react-test-renderer';
import ListHeading from './ListHeading';

describe('ListHeading', () => {
    it('matches the snapshot', () => {
        const listActions = create(<ListHeading />);

        expect(listActions.toJSON()).toMatchSnapshot();
    });
});
