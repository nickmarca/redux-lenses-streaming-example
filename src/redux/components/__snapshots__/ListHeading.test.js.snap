// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListHeading matches the snapshot 1`] = `
<div
  className="panel-block"
  style={
    Object {
      "borderTop": 0,
    }
  }
>
  <div
    className="control"
  >
    Number of messages: 
  </div>
</div>
`;
